examples/chopsticks/README.md

```diff
@@ -1,63 +1,23 @@
 # Bodhi Examples - Chopsticks
-This example demonstrates how to use [chopsticks](https://github.com/AcalaNetwork/chopsticks) to fork a substrate network, override it's storage, and test EVM+ transactions in parallel reality.
+This example demonstrates how to use [chopsticks](https://github.com/AcalaNetwork/chopsticks) to fork Acala mainnet, override it's storage, and test EVM+ transactions in parallel reality.
 
-## Setup
-### prepare storage override
-One advantage of chopsticks is that we can override storage to test more conveniently. 
-
-Suppose we want to impersonate a big whale calling a contract, we can achieve this by doing the following storage overrides:
-- set Alice's balance to `1000000000000000`, whose address is `5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY`.
-- bind a ETH address to Alice, so Alice's balance will sync to it, now this ETH address becomes the whale! The random eth address we choose here is `0xEE1b6e72FC5bC8738150B6bE7564DA887723cCA1` whose private key is `0x8d2d614677b99ee1809eec0967d538f43d3f410e20ee5f5b979dd21d5930d3fe`.
-
-Create a `storage.json` that looks like this:
-```json
-{
-  "System": {
-    "Account": [
-      [
-        ["5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY"],
-        {
-          "data": { "free": 1000000000000000 }
-        }
-      ]
-    ]
-  },
-  "EvmAccounts": {
-    "EvmAddresses": [
-      [
-        ["5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY"],
-        "0xEE1b6e72FC5bC8738150B6bE7564DA887723cCA1"
-      ]
-    ],
-    "Accounts": [
-      [
-        ["0xee1b6e72fc5bc8738150b6be7564da887723cca1"],
-        "5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY"
-      ]
-    ]
-  }
-}
-```
-
-### run chopsticks
-run the following command in the same folder with `storage.json`
+## Setup local acala fork with chopsticks
+- run a local fork of Acala (use `npx` or `yarn dlx` if you prefer these over `bunx`)
 ```
-yarn dlx @acala-network/chopsticks dev \
-  --endpoint wss://mandala-rpc.aca-staging.network/ws \
-  --import-storage=storage.json
+bunx @acala-network/chopsticks@latest -c https://raw.githubusercontent.com/AcalaNetwork/chopsticks/master/configs/acala.yml
```

Comment:
-c acala will load this file from github

Reply:
oh cool will fix, I though it loads config locally lol